ts/Series/Flags/FlagsPoint.ts

```diff
@@ -90,6 +93,16 @@ class FlagsPoint extends ColumnSeries.prototype.pointClass {
     }
 }
 
+/* *
+ *
+ *  Class Prototype
+ *
+ * */
+
+extend(FlagsPoint.prototype, {
+    ttBelow: false
+});
```

Comment:
Instead of extending the prototype, we could just add this to the class definition, around line 64:

`public ttBelow = false;`